linux_os/guide/services/ssh/sshd_approved_macs.var

```diff
@@ -11,4 +11,8 @@ operator: equals
 interactive: false
 
 options:
+{{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+    default: hmac-sha2-512,hmac-sha2-256
+{{% else %}}
     default: hmac-sha2-512,hmac-sha2-256,hmac-sha1,hmac-sha1-etm@openssh.com,hmac-sha2-256-etm@openssh.com,hmac-sha2-512-etm@openssh.com
+{{% endif %}}
```

Comment:
The current default is NIST FIPS-approved ciphers which are mandated by Federal Law. Add a new option here and don't use a duplicative default.